entrypoint

```diff
@@ -53,7 +53,6 @@ web-server/cleanup-expire-access-tokens) cd services/web-server && exec node src
 web-server/write-docs) cd services/web-server && exec node src/main.js writeDocs;;
 worker-manager/web) cd services/worker-manager && exec node src/main.js server;;
 worker-manager/provisioner) cd services/worker-manager && exec node src/main.js runProvisioner;;
-worker-manager/workerscanner) cd services/worker-manager && exec node src/main.js workerScanner;;
```

Comment:
Just to check -- this doesn't warrant a major release because removal of the k8s deployment is included in the Helm scripts -- right?

Reply:
At least in our dev env helm seems to remove the old deployment. I will check with cloudops to make sure their stuff works the same way.

Reply:
I've added a note to the changelog that they should verify this but it doesn't really feel like a breaking change.  We will probably get bitten by this at this rate :p